client/state/oauth2-clients/ui/selectors.js

```diff
@@ -34,4 +34,6 @@ export const getCurrentOAuth2Client = ( state ) => {
  * @param  {object}   state  Global state tree
  * @returns {boolean}         Whether the OAuth2 layout should be used.
  */
-export const showOAuth2Layout = ( state ) => !! getCurrentOAuth2ClientId( state );
+export const showOAuth2Layout = ( state ) => {
+	return !! getCurrentOAuth2ClientId( state );
```

Comment:
Why the change here?

Reply:
I used this code earlier to do some partner page switching  but since removed, didn't revert the code.